bcipy/helpers/tests/test_convert.py

```diff
@@ -400,7 +392,7 @@ def test_pyedf_convert_with_pre_filter(self):
     def test_pyedf_convert_with_write_targetness(self):
         """Test the pyedf_convert function with targetness for event annotations"""
         data, channels, fs, events, annotation_channels, _ = pyedf_convert(
-            self.temp_dir, write_targetness=True)
+            self.temp_dir, write_targetness=True, remove_pre_fixation=True)
```

Comment:
Do the demos need to be updated with this?

Reply:
They still seemed to run okay.